README.md

```diff
@@ -112,6 +112,36 @@ To enable UBSAN (Undefined Behaviour Sanitizer), use the following config option
    --enable-ubsan         Enable undefined behaviour checks with UBSAN
 ```
 
+To enable memory access checks with ASAN (disabled by default), use
+the following config option:
+
+```
+   --enable-asan           Enable ASAN memory access checks
+```
+
+In case plugin is configured with `--enable-asan`, the user should run
+the application with `LD_PRELOAD=<path to libasan.so>` since plugin is
```

Comment:
I don't follow this comment; normal dynamic loading rules should work, no?

Reply:
It does not seem like it will work. Without preload, we get
```
ASan runtime does not come first in initial library list; you should either link runtime to your application or manually preload it with LD_PRELOAD.
```
This approach is recommended in case app does not link ASAN and cannot be recompiled https://github.com/google/sanitizers/wiki/AddressSanitizerAsDso

Linking statically does not seem to work (https://gcc.gnu.org/bugzilla/show_bug.cgi?id=64234), either.